controllers/spannerautoscaler_controller.go

```diff
@@ -569,6 +569,14 @@ func calcDesiredPURange(sa spannerv1beta1.SpannerAutoscaler) (int, int, bool) {
 	desiredMin += sa.Spec.ScaleConfig.ProcessingUnits.Min
 	desiredMax += sa.Spec.ScaleConfig.ProcessingUnits.Max
 
+	if remainder := desiredMin % 1000; desiredMin > 1000 && remainder != 0 {
+		desiredMin = ((desiredMin / 1000) + 1) * 1000
+	}
+
+	if remainder := desiredMax % 1000; desiredMax > 1000 && remainder != 0 {
+		desiredMax = ((desiredMax / 1000) + 1) * 1000
+	}
+
```

Comment:
This solves the incremental PU issue - https://github.com/mercari/spanner-autoscaler/pull/78#issuecomment-1085492180

Reply:
LGTM!